volumina/utility/signalingDefaultDict.py

```diff
@@ -29,13 +29,13 @@ def __getitem__(self, key):        return self._dict[key]
     def __iter__(self):                return self._dict.__iter__()
     def __contains__(self, key):       return self._dict.__contains__(key)
     def get(self, key, default=None):  return self._dict.get(key, default)
-    def viewkeys(self):                return self._dict.viewkeys()
-    def items(self):                   return self._dict.items()
-    def iteritems(self):               return self._dict.iteritems()
-    def keys(self):                    return self._dict.keys()
-    def iterkeys(self):                return self._dict.iterkeys()
-    def itervalues(self):              return self._dict.itervalues()
-    def values(self):                  return self._dict.values()
+    def viewkeys(self):                return self._dict.keys()
+    def items(self):                   return list(self._dict.items())
+    def iteritems(self):               return iter(self._dict.items())
+    def keys(self):                    return list(self._dict.keys())
+    def iterkeys(self):                return iter(self._dict.keys())
+    def itervalues(self):              return iter(self._dict.values())
+    def values(self):                  return list(self._dict.values())
```

Comment:
This is again an automatic change by python utility - 2to3.
The change looks necessary to me from the principles of py3 stated in PEP469.
 See http://legacy.python.org/dev/peps/pep-0469/#id8 which says:
"In Python 3, the objects returned by d.keys(), d.values() and d. items() provide a live view of the current state of the underlying object, rather than taking a full snapshot of the current state as they did in Python 2. This change is safe in many circumstances, but does mean that, as with the direct iteration API, it is necessary to avoid adding or removing keys during iteration, in order to avoid encountering the following error..."